src/components/FilterChecklist.vue

```diff
@@ -41,14 +56,22 @@ export default {
     toggleFilterVisibility() {
       this.filtersVisible = !this.filtersVisible;
     },
+    isMobile() {
```

Comment:
Please use Bulma's responsive helpers instead https://bulma.io/documentation/modifiers/responsive-helpers/.

These scripts to detect mobile browsers are very fragile and don't always work

Reply:
Noted and Done @brenoferreira